app/views/houses/show.html.erb

```diff
@@ -1,49 +1,51 @@
 <%= render :partial => 'shared/navbar' %>
+<%= link_to "Go back", houses_path, class: 'btn my-2 my-sm-0 go-back' %>
 <div class="container">
-<div class="house-pic-card">
-  <h2>
-    <%= @house.name %>
-  </h2>
-  <%# make sure to keep the if else tag for the picture %>
-  <% if @house.photo.nil? %>
-    <img src="https://picsum.photos/400/300" alt="totally a house">
-  <% else %>
-    <%= cl_image_tag @house.photo.key, height: 300, width: 400, crop: :fill %>
-  <% end %>
-</div>
-
-
-<div class="house-details">
-  <ul>
-    <li>
-      <%= @house.address %>
-    </li>
-    <li>
-      Haunting level: <%= @house.haunted %>
-    </li>
-  </ul>
-</div>
-
-
-
-
-  <%# if/else to show booking form or update and delete buttons depending on user %>
-  <% if @house.user == current_user %>
-    <%= link_to 'Delete', house_path(@house), method: :delete, class: 'btn btn-danger btn-sm rounded-0' , data: { confirm: 'Sure?' } %>
-    <%= link_to "Update", edit_house_path(@house), class: 'btn btn-primary btn-sm rounded-0' , data: { confirm: 'Sure?' } %>
-  <% else %>
-    <div clas="booking-form">
-      <h3>
-        Please enter your preferred dates:
-      </h3>
-      <%= simple_form_for [@house, @booking] do |f| %>
-        <%= f.input :start_time, as: :date, html5: true %>
-        <%= f.input :end_time, as: :date, html5: true %>
-        <%= f.submit class: 'btn my-2 my-sm-0' %>
+      <h2 class="house-title">
+        <%= @house.name %>
+      </h2>
+<div class="row">
+  <div class="col-6 house-pic">
+    <div class="house-pic-card">
+      <%# make sure to keep the if else tag for the picture %>
+      <% if @house.photo.nil? %>
```

Comment:
sorry, I might've messed up when I originally wrote this, it should be if @house.photo.key.nil? any way you could write that in quickly?

Reply:
Done! Please check :)